provisioning/transport/http/readme.md

```diff
@@ -0,0 +1,16 @@
+#azure-iot-provisioning-http
+Communicate with Azure Device Provisioning Service over HTTP
+
+[![npm version](https://badge.fury.io/js/azure-iot-provisioning-http.svg)](https://badge.fury.io/js/azure-iot-provisioning-http)
+
+## Install
+
+`npm install -g azure-iot-provisioning-http@latest` to get the latest (pre-release) version.
```

Comment:
>latest [](start = 44, length = 6)

maybe will be revisited later but not sure if we'll use latest or something else for the tag